docs/getting-started/elements/introduction.md

```diff
@@ -0,0 +1,28 @@
+# Elements
+
+Elements provides a component for embedding API reference documentation into any existing CMS, website or React application. Here are some of the features it supports:
+
+- OpenAPI v2 (Swagger), v3.0, v3.1 and beyond
+- Interactive "try it" (API Console)
+- Beautiful display of complex schemas, requests and responses
+- Embeddable
+- Multiple layouts
+
+Check out the demo for a live example https://elements-demo.stoplight.io.
+
+> Elements can be used on your local machine and in a production environment completly for free, no strings attached.
+
+## Next steps
+
+Integrating Elements into any project is a case of loading up some JavaScript and CSS, and embedding a Web Component (custom HTML) into the HTML of whatever existing documentation you have. 
```

Comment:
> Integrate Elements into your project is simple - just add script and css tags to load elements, and then render the Elements Web Component (custom HTML) on your page where you'd like the documentation to display.

Although, this article is not specific to web components right? 🤔

Reply:
Removed it bdd9ff54b1a2434dc52989bae4f17c194597fc8d